workers/unity/ProjectSettings/ProjectSettings.asset

```diff
@@ -150,7 +150,8 @@ PlayerSettings:
   resolutionScalingMode: 0
   androidSupportedAspectRatio: 1
   androidMaxAspectRatio: 2.1
-  applicationIdentifier: {}
+  applicationIdentifier:
+    Android: com.improbable.gdk
```

Comment:
while you are on that. could you also fix that for the other platforms? I believe the standalone and ios player settings use the default one `com.Company.ProductName`....

should really change that 😂

Reply:
(y)